docs/Reference/Composable-OS-Image.md

```diff
@@ -156,6 +158,20 @@ device on top of a data device.
 | `uncompressedSize` | number | 1.0      | Yes (since 1.0) | Size of the raw uncompressed image in bytes.                                              |
 | `sha384`           | string | 1.0      | Yes (since 1.1) | SHA-384 hash of the compressed hash image.                                                |
 
+##### `Partition` Object
```

Comment:
Out of interest, why did you split out this information into a separate `partition` object?